internal/controller/ols_app_server_assets_test.go

```diff
@@ -245,7 +245,7 @@ var _ = Describe("App server assets", func() {
 			}))
 			Expect(dep.Spec.Template.Spec.Containers[1].Image).To(Equal(rOptions.LightspeedServiceImage))
 			Expect(dep.Spec.Template.Spec.Containers[1].Name).To(Equal("lightspeed-service-user-data-collector"))
-			Expect(dep.Spec.Template.Spec.Containers[0].Resources).ToNot(BeNil())
+			Expect(dep.Spec.Template.Spec.Containers[1].Resources).ToNot(BeNil())
```

Comment:
shouldn't we expect both containers(apiserver and collector) to have non-nil resources?

Reply:
The application container has been checked already at [line 210](https://github.com/openshift/lightspeed-operator/pull/169/files#diff-3125771f1a7fef7f30b28ad2a5195cbf160890fb0e79187474ebe7dab08fe60aR210). This group of checks should be for data collector container.

Reply:
ah, got it.